src/engines/jade/gui/main/main.cpp

```diff
@@ -125,6 +127,9 @@ void MainMenu::callbackActive(Widget &widget) {
 		playVideo("creditmovie");
 		return;
 	}
+
+  if (widget.getTag() == "EXIT")
+    EventMan.requestQuit();
```

Comment:
Please, indent with tabs. Tabs for indentation, spaces for alignment.

It helps if you set your editor/IDE to show tabs and/or spaces. This is how it looks in my vim setup:
![tabs](https://user-images.githubusercontent.com/156130/40681695-2b79a54c-638a-11e8-8d79-4f61b13dbece.png)

Reply:
Derp I usually pay attention to that sort of thing >.<